core/src/main/scala/tfr/instances/PredictionInstances.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package tfr.instances
+
+import cats.Show
+import com.google.protobuf.util.JsonFormat
+import io.circe.{Encoder, Printer => CircePrinter}
+import tensorflow.serving.PredictionLogOuterClass.PredictionLog
+
+trait PredictionInstances
+    extends PredictionLogShowInstances
+    with PredictionLogEncoderInstances
+
+trait PredictionLogShowInstances {
+  implicit def showPredictionLog(implicit
+      encoder: Encoder[PredictionLog]
+  ): Show[PredictionLog] =
+    new Show[PredictionLog] {
+      private[this] val Printer =
+        CircePrinter.noSpaces.copy(dropNullValues = true)
+
+      override def show(t: PredictionLog): String =
+        Printer.print(Encoder[PredictionLog].apply(t))
+    }
+}
+
+trait PredictionLogEncoderInstances extends ProtobufEncoderInstances {
+  import io.circe._
+  import io.circe.parser._
+
+  private val ProtoPrinter =
+    JsonFormat.printer().omittingInsignificantWhitespace()
+
+  implicit val predictionLogEncoder: Encoder[PredictionLog] =
+    new Encoder[PredictionLog] {
+      override def apply(a: PredictionLog): Json =
+        parse(ProtoPrinter.print(a)).getOrElse(Json.fromString(""))
```

Comment:
for now, I used `JsonFormat` for parsing the json object since I didn't want to write an `Encoder` for each new type.

Reply:
Hmm, I see, yeah in the future it would be nice to have control over those, but not sure if it's worth it right now.

Would we it make sense to have `flat` feature over these?

Reply:
I can't think of any right now, there might be a need for flattening the inputs in the PredictionLong, but seems to me like we might be forcing it a bit